### ISSUE TITLE
Couchbase Extension doesn't work in Github Codespace

### ISSUE THREAD

**Issue by shivaylamba** [couchbaselabs/VSCode-Couchbase#366]
**Describe the bug**
Couchbase Extension for VS Code doesn't work in Github Codespace

**Steps to reproduce**
Start a new Github Codespace, install Couchbase extension, and try to connect to a DB or try to enable Capella IQ. It doesn't work. 

<img width="1410" alt="image" src="https://github.com/couchbaselabs/VSCode-Couchbase/assets/19529592/24d926d2-f7c0-4220-ab34-af209c7b57b8">

**Comment by lokesh-couchbase** [comment]
So I tested our extension on Codespaces and I also faced same issue

Updated the build to use same version of linux as in github codespaces and this seems to make it work. 
We can use iQ.

However, I am still not able to make connection tree working as our extension require OS keychain access. This however requires display which is not available on Codespaces. 

One possible work around is having a separate build for codespaces, codeready and other cloud VSCode platforms. @deniswsrosa thoughts on this?
